resolwe_bio/processes/clustering/find_simmilar.py

```diff
@@ -0,0 +1,207 @@
+"""Find genes with similar expression profile."""
+import json
+from collections import defaultdict
+
+import pandas as pd
+from scipy.stats import pearsonr, spearmanr
+
+from resolwe.process import (
+    DataField,
+    JsonField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+def get_expression(fname, sep="\t"):
+    """Read expressions from file."""
+    df = pd.read_csv(
```

Comment:
We use `read_csv` to read in expression profiles in many process. We should specify this function somewhere and import it everywhere else. Feel free to leave this as it is and I'll talk to @jkokosar about it.